tests/system-tests/rdscore/internal/rdscorecommon/sriov-pod-level-bond.go

```diff
@@ -0,0 +1,1114 @@
+	}
+
+	glog.V(100).Infof("Ensure %s deployment does not exist in namespace %s", deploymentName, nsName)
+
+	err = apiobjectshelper.DeleteDeployment(apiClient, deploymentName, nsName)
+
+	if err != nil {
+		glog.V(100).Infof("Failed to delete deployment %s from nsname %s due to %v",
+			deploymentName, nsName, err)
+
+		return fmt.Errorf("failed to delete deployment %s from nsname %s due to %w",
+			deploymentName, nsName, err)
+	}
+
+	err = apiobjectshelper.EnsureAllPodsRemoved(APIClient, nsName, podLabel)
+
+	if err != nil {
+		glog.V(100).Infof("Failed to delete pods in namespace %s with the label %s: %w", nsName, podLabel, err)
+
+		return fmt.Errorf("failed to delete pods in namespace %s with the label %s: %w", nsName, podLabel, err)
+	}
+
+	return nil
+}
+
+func definePodLevelBondDeploymentContainer() *pod.ContainerBuilder {
+	cName := "test-pod"
+
+	containerCmd := []string{
+		"testcmd",
+		"--listen",
+		"-interface",
+		"net3",
+		"-protocol",
+		"tcp",
+		"-port",
+		RDSCoreConfig.PodLevelBondPort,
+	}
+
+	glog.V(100).Infof("Creating container %q", cName)
+
+	deploymentContainer := pod.NewContainerBuilder(cName, RDSCoreConfig.PodLevelBondDeployImage, containerCmd)
+
+	glog.V(100).Infof("Defining SecurityContext")
+
+	var trueFlag = true
+
+	userUID := new(int64)
+
+	*userUID = 0
+
+	securityContext := &corev1.SecurityContext{
+		RunAsUser:  userUID,
+		Privileged: &trueFlag,
+		SeccompProfile: &corev1.SeccompProfile{
+			Type: corev1.SeccompProfileTypeRuntimeDefault,
+		},
+		Capabilities: &corev1.Capabilities{
+			Add: []corev1.Capability{"NET_RAW", "NET_ADMIN", "SYS_ADMIN", "IPC_LOCK"},
+		},
+	}
+
+	glog.V(100).Infof("Setting SecurityContext")
+
+	deploymentContainer = deploymentContainer.WithSecurityContext(securityContext)
+
+	glog.V(100).Infof("Dropping ALL security capability")
+
+	deploymentContainer = deploymentContainer.WithDropSecurityCapabilities([]string{"ALL"}, true)
+
+	glog.V(100).Infof("%q container's  definition:\n%#v", cName, deploymentContainer)
+
+	return deploymentContainer
+}
+
+func definePodLevelBondTestPodDeployment(
+	apiClient *clients.Settings,
+	containerConfig *corev1.Container,
+	deploymentName,
+	nsName,
+	scheduleOnHost,
+	sriovNet1Name,
+	sriovNet2Name,
+	bondNetName,
+	bondInfIPv4,
+	bondInfIPv6,
+	bondInfSubMaskIPv4,
+	bondInfSubMaskIPv6,
+	bondInfMacAddr string,
+	deployLabels map[string]string) *deployment.Builder {
+	glog.V(100).Infof("Defining deployment %q in %q ns", deploymentName, nsName)
+
+	nodeSelector := map[string]string{"kubernetes.io/hostname": scheduleOnHost}
+
+	netAnnotations := []*types.NetworkSelectionElement{
+		{
+			Name:      sriovNet1Name,
+			Namespace: nsName,
+		},
+		{
+			Name:      sriovNet2Name,
+			Namespace: nsName,
+		},
+		{
+			Name:      bondNetName,
+			Namespace: nsName,
+			IPRequest: []string{fmt.Sprintf("%s/%s", bondInfIPv4, bondInfSubMaskIPv4),
+				fmt.Sprintf("%s/%s", bondInfIPv6, bondInfSubMaskIPv6)},
```

Comment:
It worth validating input, for e.g. checking that both provided params ( IP address and subnet) ain't empty.
Also what about a case when only IPv4 or IPv6 pair is passed?